[2024-06-22T07:39:03.846+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-22T07:39:03.856+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag_v2.first_task manual__2024-06-22T07:37:50.061876+00:00 [queued]>
[2024-06-22T07:39:03.860+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag_v2.first_task manual__2024-06-22T07:37:50.061876+00:00 [queued]>
[2024-06-22T07:39:03.860+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 7
[2024-06-22T07:39:03.866+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): first_task> on 2024-06-22 07:37:50.061876+00:00
[2024-06-22T07:39:03.869+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1957) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-22T07:39:03.870+0000] {standard_task_runner.py:63} INFO - Started process 1958 to run task
[2024-06-22T07:39:03.870+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'first_dag_v2', 'first_task', 'manual__2024-06-22T07:37:50.061876+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp5jf4yawo']
[2024-06-22T07:39:03.871+0000] {standard_task_runner.py:91} INFO - Job 12: Subtask first_task
[2024-06-22T07:39:03.889+0000] {task_command.py:426} INFO - Running <TaskInstance: first_dag_v2.first_task manual__2024-06-22T07:37:50.061876+00:00 [running]> on host 65f8df8d6172
[2024-06-22T07:39:03.925+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gnosia93' AIRFLOW_CTX_DAG_ID='first_dag_v2' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-22T07:37:50.061876+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-22T07:37:50.061876+00:00'
[2024-06-22T07:39:03.925+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-22T07:39:03.931+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-22T07:39:03.931+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world. gnosia93']
[2024-06-22T07:39:03.935+0000] {subprocess.py:86} INFO - Output:
[2024-06-22T07:39:03.936+0000] {subprocess.py:93} INFO - hello world. gnosia93
[2024-06-22T07:39:03.936+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-22T07:39:03.936+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-22T07:39:03.947+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=first_dag_v2, task_id=first_task, run_id=manual__2024-06-22T07:37:50.061876+00:00, execution_date=20240622T073750, start_date=20240622T073903, end_date=20240622T073903
[2024-06-22T07:39:03.975+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-22T07:39:03.985+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-22T07:39:03.985+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
